### PR TITLE
Add xc-skiing example and update examples

### PR DESCRIPTION
**Pull request by olemarius90**
All examples have been updated by Kaja, an additional example on cross-country skiing have been added by Ole Marius.

We aim to publish this example online: https://www.sintef.no/projectweb/autoactive/code-example/ and hopefully present it at ICSS 2023.

### CODE REVIEW

File: ToDo_Toolbox.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+1. Both AutoActive Matlab toolbox and AutoActive Research Environment Toolbox is used as name
```
Comment (aliverud) [inline ToDo_Toolbox.txt]: Denne fila skal kanskje ikke med @olemarius90 ?

File: MatlabToolbox/external/peakseek/peakseek.m [hunk 2]
```diff
@@ -0,0 +1,50 @@
+function [locs pks]=peakseek(x,minpeakdist,minpeakh)
+% Alternative to the findpeaks function.  This thing runs much much faster.
+% It really leaves findpeaks in the dust.  It also can handle ties between
+% peaks.  Findpeaks just erases both in a tie.  Shame on findpeaks.
+%
+% x is a vector input (generally a timecourse)
+% minpeakdist is the minimum desired distance between peaks (optional, defaults to 1)
+% minpeakh is the minimum height of a peak (optional)
+%
+% (c) 2010
```
Comment (aliverud) [inline MatlabToolbox/external/peakseek/peakseek.m:10]: @olemarius90 redistribusjon ok?

Comment (aliverud) [reply MatlabToolbox/external/peakseek/peakseek.m:10]: Ser at det er ok nå.

File: ToDo_Toolbox.txt (hunk 1 above)
Comment (olemarius90) [reply ToDo_Toolbox.txt]: Stemmer! Fjerna den nå :)

**Pull request merged**